python/tidy/servo_tidy/tidy.py

```diff
@@ -645,7 +645,9 @@ def get_file_list(directory, only_changed_files=False, exclude_dirs=[]):
         args = ["git", "ls-files", "--others", "--exclude-standard", directory]
         file_list += subprocess.check_output(args)
         for f in file_list.splitlines():
-            yield os.path.join('.', f)
+            if os.path.join('.', os.path.dirname(f)) not in ignored_dirs:
+                yield os.path.join('.', f)
+
     elif exclude_dirs:
         for root, dirs, files in os.walk(directory, topdown=True):
             # modify 'dirs' in-place so that we don't do unwanted traversals in excluded directories
```

Comment:
btw, we should also change the `ignored_dirs` in this `elif` branch to `exclude_dirs`, so that it makes sense :)